go-controller/pkg/clustermanager/routeadvertisements/controller.go

```diff
@@ -398,7 +404,7 @@ func (c *Controller) generateFRRConfigurations(ra *ratypes.RouteAdvertisements)
 	// ordered
 	slices.Sort(selectedNetworks.vrfs)
 	slices.Sort(selectedNetworks.subnets)
-	selectedNetworks.networks = sets.List(networkSet)
+	selectedNetworks.networks = networks
```

Comment:
This was previously an ordered list. With this change this will not be iterated in order. Have you crosschecked this is not an issue? The idea of having ordered things is to produce consistent FRRConfigurations and avoid unnecessary updates just because we only change the order in which we list things.